packages/manager/modules/exchange/src/account/update/account-update.controller.js

```diff
@@ -312,24 +312,26 @@ export default class ExchangeUpdateAccountCtrl {
   }
 
   setQuotaAvailable() {
-    this.newAccountOptions.quotaArray = [];
+    this.updateAccountOptions.quotaArray = [];
 
     for (
-      let i = this.newAccountOptions.maxQuota;
-      i >= this.newAccountOptions.minQuota;
+      let i = this.updateAccountOptions.maxQuota;
+      i >= this.updateAccountOptions.minQuota;
       i -= 1
```

Comment:
```suggestion
      i--
```

Reply:
I have just try to replace it.
it seems it was write like this because `i--` produce eslint error message.
So let it like it was write before.

Thanks